### ISSUE TITLE
🤖 CREATE SOUND CLASS💪😎🤳

### ISSUE THREAD

**Issue by corners2wall** [corners2wall/Skyscraper#8]
Hello everyone! This repo needs a new entity 👷💥💫

Description: in future we will add sounds on different user events for better user experience. 💯📈

What should be done?
- Add new class `Sound` in `Components` folder; 🎸🎷🎻
- Sound should be implement next interface 
```typescript
interface Sound {
  play(): void; // play sound 
  stop(): void; // stop sound 
  repeat(count: number ): void; // repeat sound `count` once or infinity 
  setVolume(value: number): void // set volume of sound 
}

const gameSound = new Sound('path/to/sound');
```
P.S. you can be creativity and add more methods. 

JUST DO IT 💪💪💪
Good luck and have fun! 🧡

**Comment by Vidhanvyrs** [comment]
hey @corners2wall i think i can help u with this issue can you please assign it to me so that i  can start working on it rightaway

**Comment by corners2wall** [comment]
DON'T STOP DUDE. DO IT

**Comment by Vidhanvyrs** [comment]
hey @corners2wall i have added a PR related to this issue pleas check it out

**Comment by corners2wall** [comment]
Closes #13
